.mailmap

```diff
@@ -11,7 +11,7 @@ Mainak Jas <mainakjas@gmail.com> <jasmainak@users.noreply.github.com>
 Mainak Jas <mainakjas@gmail.com> <mainak.jas@telecom-paristech.fr>
 Matthew Antalek <matthew.antalek@gmail.com> Matthew Antalek <matthew.antalek@northwestern.edu>
 Matthew Antalek <matthew.antalek@gmail.com> Matt Antalek <matthew.antalek@gmail.com>
-Mayank Agrawal <magrawa1@swarthmore.edu>
+Mayank Agrawal <mayankagrawal96@gmail.com> Mayank Agrawal <magrawa1@swarthmore.edu>
```

Comment:
```suggestion
Mayank Agrawal <mayankagrawal96@gmail.com> timshell <magrawa1@swarthmore.edu>
```

Earlier, we were just associating the old email with a new name but now since both the commit email and name has changed, you need to give both the old name and the old email.